### ISSUE TITLE
Will we support YOLOV5 model in the future?

### ISSUE THREAD

**Issue by duany049** [microsoft/SoftTeacher#31]

**Comment by MendelXu** [comment]
Currently, we have no plan to add support to YOLO.

**Comment by duany049** [comment]
> Currently, we have no plan to add support to YOLO.

Because of work requirements,  I have plan to add support to YOLOV5, If my code works well, could I submit the code ?

**Comment by MendelXu** [comment]
Yes. Also, if you face any problems when adapting it to YOLOV5, it is my pleasure to assist you.

**Comment by duany049** [comment]
> Yes. Also, if you face any problems when adapting it to YOLOV5, it is my pleasure to assist you.

Thank you very much.

**Comment by Jack-Hu-2001** [comment]
I'm working on similar things(not YOLOv5 exactly), I comment here to make myself informed with newest messages.

**Comment by Fly-dream12** [comment]
What can be wrong if the pseudo loss is always 0 in the training stage when i implemented it in YOLOv5 structure，can you give some suggestions for me ？@duany049 @MendelXu
